modules/mod_custom/src/Dispatcher/Dispatcher.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
```

Comment:
```suggestion
<?php

/**
```
Fix PHPCS failing in drone.